docs/en_US/NAS/BenchmarksExample.ipynb

```diff
@@ -1,379 +1,366 @@
+      "execution_count": 9,
+      "metadata": {
+        "tags": []
+      },
+      "outputs": [],
+      "source": [
+        "# count number\n",
+        "model_spec = {'num_nodes_normal': 5, 'num_nodes_reduce': 5, 'depth': 12, 'width': 32, 'aux': False, 'drop_prob': 0.0}\n",
+        "cell_spec = {\n",
+        "    'normal_0_op_x': 'avg_pool_3x3',\n",
+        "    'normal_0_input_x': 0,\n",
+        "    'normal_0_op_y': 'conv_7x1_1x7',\n",
+        "    'normal_0_input_y': 1,\n",
+        "    'normal_1_op_x': 'sep_conv_3x3',\n",
+        "    'normal_1_input_x': 2,\n",
+        "    'normal_1_op_y': 'sep_conv_5x5',\n",
+        "    'normal_1_input_y': 0,\n",
+        "    'normal_2_op_x': 'dil_sep_conv_3x3',\n",
+        "    'normal_2_input_x': 2,\n",
+        "    'normal_2_op_y': 'dil_sep_conv_3x3',\n",
+        "    'normal_2_input_y': 2,\n",
+        "    'normal_3_op_x': 'skip_connect',\n",
+        "    'normal_3_input_x': 4,\n",
+        "    'normal_3_op_y': 'dil_sep_conv_3x3',\n",
+        "    'normal_3_input_y': 4,\n",
+        "    'normal_4_op_x': 'conv_7x1_1x7',\n",
+        "    'normal_4_input_x': 2,\n",
+        "    'normal_4_op_y': 'sep_conv_3x3',\n",
+        "    'normal_4_input_y': 4,\n",
+        "    'normal_concat': [3, 5, 6],\n",
+        "    'reduce_0_op_x': 'avg_pool_3x3',\n",
+        "    'reduce_0_input_x': 0,\n",
+        "    'reduce_0_op_y': 'dil_sep_conv_3x3',\n",
+        "    'reduce_0_input_y': 1,\n",
+        "    'reduce_1_op_x': 'sep_conv_3x3',\n",
+        "    'reduce_1_input_x': 0,\n",
+        "    'reduce_1_op_y': 'sep_conv_3x3',\n",
+        "    'reduce_1_input_y': 0,\n",
+        "    'reduce_2_op_x': 'skip_connect',\n",
+        "    'reduce_2_input_x': 2,\n",
+        "    'reduce_2_op_y': 'sep_conv_7x7',\n",
+        "    'reduce_2_input_y': 0,\n",
+        "    'reduce_3_op_x': 'conv_7x1_1x7',\n",
+        "    'reduce_3_input_x': 4,\n",
+        "    'reduce_3_op_y': 'skip_connect',\n",
+        "    'reduce_3_input_y': 4,\n",
+        "    'reduce_4_op_x': 'conv_7x1_1x7',\n",
+        "    'reduce_4_input_x': 0,\n",
+        "    'reduce_4_op_y': 'conv_7x1_1x7',\n",
+        "    'reduce_4_input_y': 5,\n",
+        "    'reduce_concat': [3, 6]\n",
+        "}\n",
+        "\n",
+        "for t in query_nds_trial_stats('nas_cell', None, None, model_spec, cell_spec, 'cifar10'):\n",
+        "    assert t['config']['model_spec'] == model_spec\n",
+        "    assert t['config']['cell_spec'] == cell_spec\n",
+        "    pprint.pprint(t)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 10,
+      "metadata": {
+        "tags": []
+      },
+      "outputs": [],
+      "source": [
+        "# count number\n",
+        "print('NDS (amoeba) count:', len(list(query_nds_trial_stats(None, 'amoeba', None, None, None, None, None))))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## NLP"
```

Comment:
also suggest to add a figure to show the NLP search space, similar to the other three benchmarks

Reply:
ok, let me check that

Reply:
done